app/assets/stylesheets/_variables.scss

```diff
@@ -37,8 +37,6 @@ $bg_color_flash_error: #FFDDDD !default; // error
 $font_color_flash_error: $colors_black !default;
 $bg_color_flash_success: #DDEEDD !default; // success
 $font_color_flash_success: $colors_black !default;
-$bg_color_flash_inbox: #DDDDEE !default; // new message
-$font_color_flash_inbox: $colors_black !default;
```

Comment:
What's up with `.flash.inbox` being removed? (Here and everywhere else?)

Reply:
We don't use it anymore? It was for the new message notifications we used to get in the flash format but now have the button thingy for

Reply:
OH. Right, yes, we don't use it any more, cool.